.cspell.json

```diff
@@ -0,0 +1,38 @@
+{
+  "version": "0.2",
+  "words": [
+    "abcxyz",
+    "addrs",
+    "anotherhashishere",
+    "arcanis",
+    "Builtins",
+    "complexm",
+    "endregion",
+    "entrypoints",
+    "filesystems",
+    "hashishere",
+    "Hierachic",
```

Comment:
This is actually a typo in ModulesInHierachicDirectoriesPlugin, but will be a breaking change to fix